data/cc-agent.target

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Clear Containers Agent Target
+Requires=basic.target
+Requires=cc-agent.service
+Conflicts=rescue.service rescue.target
+After=basic.target rescue.service rescue.target
+AllowIsolate=yes
```

Comment:
I dont think that it is really needed, it allows to use "systemctl isolate" to change a current target to this target, but it is recommended to used for systemd targets:
 "It probably is a good idea to leave this disabled except for target units that shall be used similar to runlevels in SysV init systems, just as a precaution to avoid unusable system states."
https://www.freedesktop.org/software/systemd/man/systemd.unit.html#AllowIsolate=